file_api/views.py

```diff
@@ -44,6 +45,13 @@ def pretty_request(request):
 @require_http_methods(["POST"])
 @csrf_exempt
 def new_file(request):
+    if int(request.GET.get('expense', '0')) > int(request.GET.get('invoice', '0')):
```

Comment:
Feels like this could be done better.

Reply:
Merged them together. Please do check again if you think I did it in a reasonable way

Reply:
Looks nice, seems to work